.gitmodules

```diff
@@ -1,6 +1,8 @@
 [submodule "qemu"]
 	path = qemu
-	url = https://github.com/pranith/qemu
+	url = https://github.com/B-Bimmermann/qemu-1.git
+	branch = qsim_v25_qemu_26
```

Comment:
At the moment I think can't do it.
I think I need it for the merge with the xilinx qemu : https://github.com/Xilinx/qemu
Is there a better way to do this ?

But you can change it every time for your repro.